tools/statvar_importer/config_flags.py

```diff
@@ -0,0 +1,522 @@
+        'dc_api_use_cache':
+            False,
+        'dc_api_batch_size':
+            100,
+        # Settings from flags
+        'pv_map':
+            _FLAGS.pv_map,
+        'input_rows':
+            _FLAGS.input_rows,
+        'input_columns':
+            _FLAGS.input_columns,
+        'skip_rows':
+            _FLAGS.skip_rows,
+        'header_rows':
+            _FLAGS.header_rows,
+        'header_columns':
+            _FLAGS.header_columns,
+        'process_rows': [0],
+        'parallelism':
+            _FLAGS.parallelism,
+        'output_counters':
+            _FLAGS.output_counters,
+
+        # Settings for spell checks
+        'spell_check':
+            _FLAGS.spell_check,
+        'spell_allowlist':
+            _FLAGS.spell_check_allow_list,
+        'spell_allow_words': [],
+        'output_sanity_check':
+            _FLAGS.sanity_check_output,
+        'spell_check_text_only':
+            _FLAGS.spell_check_text,
+        'spell_check_ignore_props':
+            _FLAGS.spell_check_ignore_props,
+        'debug':
+            _FLAGS.debug,
+        'log_level':
+            _FLAGS.log_level,
+
+        # Settings for PV Map generator
+        'generate_pvmap':
+            _FLAGS.generate_pvmap,
+        'google_api_key':
+            _FLAGS.google_genai_key,
+        'sample_pvmap':
+            _FLAGS.sample_pvmap,
+        'sample_statvars':
+            _FLAGS.sample_statvars,
+        'data_context':
+            _FLAGS.data_context,
+        'llm_data_annotation':
+            _FLAGS.generate_pvmap,
+
+        # Settings for statvar name generator
+        'generate_statvar_name':
+            _FLAGS.generate_statvar_name,  # Generate names for StatVars
+        'llm_generate_statvar_name':
+            _FLAGS.llm_generate_statvar_name,
+    }
+
+
+def get_config_from_flags(filename: str = None) -> ConfigMap:
+    """Returns a Config object with parameters loaded from a file.
+
+  Args:
+    filename: name of the file to load.
+
+  Returns:
+    Config object with all the parameters loaded into the config_dict.
+  """
+    config_dict = dict(get_default_config())
+    if isinstance(filename, dict):
+        config_dict.update(filename)
+        filename = None
+    elif isinstance(filename, ConfigMap):
+        config_dict.update(filename.get_configs())
+    elif isinstance(filename, str):
+        file_config = {}
+        # Check if filename is a file.
+        config_files = file_util.file_get_matching(filename)
+        if config_files:
+            # Load config from file.
+            file_config = file_util.file_load_py_dict(config_files)
+        elif ':' in filename:
+            # Try parsing config as a string
+            file_config = _parse_dict(filename)
+        if file_config:
+            update_config(file_config, config_dict)
+    else:
+        logging.error(f'Unknown config {filename}, ignored')
+    config = ConfigMap(config_dict=config_dict)
+    # Set logging verbosity
```

Comment:
Moved to a separate function and renamed this ti init_config_from_flags().

Calling verbosity here since config_flags:init_config_from_flags()  is used by multiple modules within statvar processor.